### PR TITLE
104 hero wave

### LINKED ISSUE #104: [Hero] Implemenent "Hero (wave)" version of the hero block
Some pages use a special here version with a "wave" hero styling. This should be implemented as variant of the hero block. We keep the content as close as possible to the default hero, the wave button image is always the same and added by the block.

Example: https://www.moleculardevices.com/en/assets/tutorials-videos/br/critical-considerations-for-choosing-right-microplate-reader
Franklin: https://main--moleculardevices--hlxsites.hlx.page/en/assets/tutorials-videos/br/critical-considerations-for-choosing-right-microplate-reader

### PR DESCRIPTION
**Pull request by paolomoz**
Fix #104

Original: https://www.moleculardevices.com/en/assets/tutorials-videos/br/critical-considerations-for-choosing-right-microplate-reader

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/tutorials-videos/br/critical-considerations-for-choosing-right-microplate-reader
- After: https://104-hero-wave--moleculardevices--hlxsites.hlx.page/en/assets/tutorials-videos/br/critical-considerations-for-choosing-right-microplate-reader

### CODE REVIEW

File: blocks/hero/hero.js [hunk 1]
```diff
@@ -102,6 +102,15 @@ export function buildHero(block) {
     block.parentElement.appendChild(container);
   }
 
+  if (block.classList.contains('wave')) {
```
Comment (lydiapuric) [inline blocks/hero/hero.js:105]: @paolomoz why didn't you levarage css for the hero wave, like we did for the footer wave?

**Comment by mhaack** [comment]
In the PR branch the hero as a thin bottom border which should be removed.

File: blocks/hero/hero.js (hunk 1 above)
Comment (paolomoz) [reply blocks/hero/hero.js:105]: @lydiapuric do you have a link to the example you are referring to?

**Comment by paolomoz** [comment]
> @paolomoz Here is the reference to the footer wave css, see https://github.com/hlxsites/moleculardevices/blob/main/styles/styles.css#L1067

@lydiapuric thank you!
Done

**Review by jose-correia — APPROVED** [review]

**Pull request merged**